dspace/config/dspace.cfg

```diff
@@ -1575,6 +1575,9 @@ request.item.helpdesk.override = false
 # ${dspace.dir}/config/ directory.
 module_dir = modules
 
+## Clarin-DSpace properties
+upload.file.delete = true
```

Comment:
consider renaming the prop to something like "delete.file.after.admin.upload" or similar to explain more what changing that flag means

Reply:
fixed